apps/condo/pages/settings/mobileFeatureConfig/onlyProgressionMeterReadings/index.tsx

```diff
@@ -0,0 +1,60 @@
+import { Col, Row, Typography } from 'antd'
+import { Gutter } from 'antd/es/grid/row'
+import get from 'lodash/get'
+import Head from 'next/head'
+import React from 'react'
+
+import { useIntl } from '@open-condo/next/intl'
+import { useOrganization } from '@open-condo/next/organization'
+
+import { PageContent, PageHeader, PageWrapper } from '@condo/domains/common/components/containers/BaseLayout'
+import {
+    OnlyProgressionMeterReadingsForm,
+} from '@condo/domains/settings/components/ticketSubmitting/OnlyProgressionMeterReadingsForm'
+import { MobileFeatureConfig } from '@condo/domains/settings/utils/clientSchema'
+
+const ROW_GUTTER: [Gutter, Gutter] = [0, 60]
+
+const OnlyProgressionMeterReadingsContent: React.FC = () => {
+    const intl = useIntl()
+    const PageTitle = intl.formatMessage({ id: 'pages.condo.settings.barItem.MobileFeatureConfig.OnlyProgressionMeterReadings.title' })
+    
+    const userOrganization = useOrganization()
+    const userOrganizationId = get(userOrganization, ['organization', 'id'])
+
+    const { obj: mobileConfig, loading } = MobileFeatureConfig.useObject({
+        where: {
+            organization: { id: userOrganizationId },
+        },
+    })
+    return (!loading &&
+        <>
+            <PageHeader title={<Typography.Title>{PageTitle}</Typography.Title>} />
+            <Row gutter={ROW_GUTTER}>
+                <Col span={24}>
+                    <OnlyProgressionMeterReadingsForm mobileConfig={mobileConfig} userOrganizationId={userOrganizationId}/>
+                </Col>
+            </Row>
+        </>
+    )
```

Comment:
```suggestion
    return !loading && (
        <>
            <PageHeader title={<Typography.Title>{PageTitle}</Typography.Title>} />
            <Row gutter={ROW_GUTTER}>
                <Col span={24}>
                    <OnlyProgressionMeterReadingsForm mobileConfig={mobileConfig} userOrganizationId={userOrganizationId}/>
                </Col>
            </Row>
        </>
    )
```

Reply:
What the user will see while `loading===true`?

Reply:
most likely a page with no content